pkg/actions/fargate/tasks.go

```diff
@@ -17,16 +20,20 @@ type createFargateStackTask struct {
 
 func (t *createFargateStackTask) Describe() string { return "create fargate IAM stacK" }
 
-func makeClusterStackName(clusterName string) string {
-	return "eksctl-" + clusterName + "-fargate"
+func makeClusterStackName(clusterName string, disableStackPrefix bool) string {
+	stackName := fmt.Sprintf("%s-fargate", clusterName)
+	if !disableStackPrefix {
+		stackName = "eksctl-" + stackName
+	}
+	return strings.Replace(stackName, "_", "-", -1)
```

Comment:
Are all these `makeXName` funcs similar enough that we can refactor them into one? With a test?

Reply:
I included a reusable func and changed these stackname funcs to use that. Added some test cases for that reusable func as well!